### PR TITLE
PEP8 fixes

### LINKED ISSUE #34: pep8 the repo, make spacing a consistent 4 spaces

### PR DESCRIPTION
**Pull request by zain**
Would love to have a second pair of eyeballs take a look at this! There's a lot of changes to indentation so I'm hoping I didn't over/under indent anything.

Fixes #34.

### CODE REVIEW

File: src/download_labels.py [hunk 1]
```diff
@@ -46,42 +48,33 @@ def way(self, w):
...
-          name = tag.v
         if tag.k == self.extract_type:
           way_type = tag.v
           should_extract = True
 
       if not should_extract:
         return
 
-      if not way_type in self.types:
+      if way_type not in self.types:
```
Comment (zain) [inline src/download_labels.py]: Note that this isn't just a style change -- this actually changes the functionality. But I'm assuming the previous behavior was a bug here.

File: src/create_training_data.py [hunk 2]
```diff
@@ -17,350 +17,353 @@
...
-  '''
-      turn on a pixel in way_bitmap if its in bounds
-  '''
-  if x < NAIP_PIXEL_BUFFER or y < NAIP_PIXEL_BUFFER or x >= len(way_bitmap[0])-NAIP_PIXEL_BUFFER or y >= len(way_bitmap)-NAIP_PIXEL_BUFFER:
-    return
-  way_bitmap[y][x] = 1
+    '''
+        turn on a pixel in way_bitmap if its in bounds
+    '''
+    if x < NAIP_PIXEL_BUFFER or y < NAIP_PIXEL_BUFFER or x >= len(way_bitmap[
```
Comment (andrewljohnson) [inline src/create_training_data.py:165]: isn't this a weird break?

File: src/download_labels.py [hunk 3]
```diff
@@ -1,143 +1,143 @@
...
+        way_type = None
+        for tag in w.tags:
+            if tag.k == self.extract_type:
+                way_type = tag.v
+                should_extract = True
+
+        if not should_extract:
+            return
+
+        if way_type not in self.types:
```
Comment (andrewljohnson) [inline src/download_labels.py:82]: Yeah, you are right, that was a bug. We weren't using the types, so it wasn't showing up.

**Comment by andrewljohnson** [comment]
This lgtm, though it's hard to review of course.

I'm running the code to make sure it seems to work too, but I'm leaving to go camping before I'll know it's all done :)

Shipping it seems fine to me... if something got busted, we can fix. @silberman might want to check in on top of the pep8ed code this weekend.

**Pull request merged**